xml/System.Net.Http.Headers/ProductInfoHeaderValue.xml

```diff
@@ -44,7 +44,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Net.Http.Headers.ProductInfoHeaderValue> class provides support for the product and comment tokens values used in  a User-Agent header as defined in [RFC 2616](https://go.microsoft.com/fwlink/?LinkID=241642) by the IETF.  
+ The <xref:System.Net.Http.Headers.ProductInfoHeaderValue> class provides support for the product and comment tokens values used in a User-Agent header as defined in [RFC 2616](https://go.microsoft.com/fwlink/?LinkID=241642) by the IETF.  
```

Comment:
Why link it as fwlink?
Also 2616 has been obsoleted, twice already, the newest RFC is here: https://www.rfc-editor.org/rfc/rfc9110#field.user-agent